aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/LambdaContainerHandler.java

```diff
@@ -215,6 +215,9 @@ public void proxyStream(InputStream input, OutputStream output, Context context)
         } catch (JsonMappingException e) {
             log.error("Error while mapping object to RequestType class", e);
             getObjectMapper().writeValue(output, exceptionHandler.handle(e));
+        } catch (Exception e) {
+            log.error("Error while proxying request.", e);
+            getObjectMapper().writeValue(output, exceptionHandler.handle(e));
```

Comment:
See comment on the PR. We should rely on the ExceptionHandler object instead like we do in the `proxy()` method.